### PR TITLE
fix(DB): Move location of Chest of The Seven in BRD

### PR DESCRIPTION
**Pull request by Zoidwaffle**
Inside Blackrock Depths the "Chest of The Seven" spawns in a slightly wrong location.

## Changes Proposed:
-  Move the location of the chest

## Issues Addressed:
- Closes [#4293 ](https://github.com/azerothcore/azerothcore-wotlk/issues/4293)

## Tests Performed:
- Apply SQL, see if location of the chest changes after a server restart
- Please see the [#4293](https://github.com/azerothcore/azerothcore-wotlk/issues/4293) issue where I have screenshots

## How to Test the Changes:

- .tele brd
- .go c id 9039
- As a GM you can see the chest not being by the door before applying the SQL query.

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1610739863539733975.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1610739863539733975');
+
+-- "Chest of The Seven" from Blackrock Depths should spawn in a different position
+UPDATE gameobject SET position_x="1265.55", position_y="-284.421", position_z="-78.2193", orientation="0.786698", rotation2="-0.383284", rotation3="-0.923631" WHERE guid=67872;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1610739863539733975.sql:4]: add backticks ( \` ) around keywords and remove the double quotes around all numbers ( " )

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1610739863539733975.sql (hunk 1 above)
Comment (Zoidwaffle) [reply data/sql/updates/pending_db_world/rev_1610739863539733975.sql]: Done!

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
thanks for your contribution @Zoidwaffle !

**Comment by Zoidwaffle** [comment]
I tried to change my name and stuff broke, I'm closing this PR... will create a new one later....

**Pull request closed without merging**